src/ordered-elements.js

```diff
@@ -19,9 +19,12 @@ export default class OrderedElements {
         }
     }
 
-    set(key /* : string */, value /* : any */) {
+    set(key /* : string */, value /* : any */, preserveOrder /* : boolean */) {
```

Comment:
Hmm, could we maybe have `preserveOrder` default to `true` and set it to `false` in all of the places we explicitly want it to override?

Reply:
We want this to be false in every place except one: string handlers. I'm happy to flip it to be `reorder` or `shouldReorder` and make sure that it is always passed in though if you think that is better.

Reply:
Alright, I added a commit that inverts this!